drivers/ethernet/eth_sam_gmac.c

```diff
@@ -857,12 +1071,10 @@ static int gmac_init(Gmac *gmac, u32_t gmac_ncfgr_val)
 
 	/* Disable all interrupts */
 	gmac->GMAC_IDR = UINT32_MAX;
-	gmac->GMAC_IDRPQ[GMAC_QUE_1 - 1] = UINT32_MAX;
-	gmac->GMAC_IDRPQ[GMAC_QUE_2 - 1] = UINT32_MAX;
 	/* Clear all interrupts */
 	(void)gmac->GMAC_ISR;
-	(void)gmac->GMAC_ISRPQ[GMAC_QUE_1 - 1];
-	(void)gmac->GMAC_ISRPQ[GMAC_QUE_2 - 1];
+	disable_all_queue_interrupt(gmac);
```

Comment:
The name of the function should actually be `disable_all_priority_queue_interrupt` since 
```
	gmac->GMAC_IDR = UINT32_MAX;
	(void)gmac->GMAC_ISR;
```
is not included in `disable_all_queue_interrupt`.

Reply:
Fixed on 821170d